diagnostics/tropical_pacific_sea_level/dynamical_balance2.py

```diff
@@ -0,0 +1,268 @@
+import numpy as np
+
+
+####
+# Input variable are all xr.DataArray
+####
+
+####
+# constant 
+r_earth = 6.371*1E8         # cm
+
+def curl_var(da_uo,da_vo,x_name='lon',y_name='lat'):
```

Comment:
Is this something that xgcm could help with?  Note how in this example they do a curl calculation https://xgcm.readthedocs.io/en/stable/example_mitgcm.html

Reply:
Right! During creating the code, I did not notice xgcm's functionality on curl. Therefore, I wrote the calculation myself. Do you want me to change to use xgcm instead?